### PR TITLE
Only pack requirements for Python runtimes

### PR DESCRIPTION
**Pull request by sbonami**

### CODE REVIEW

File: test.bats [hunk 1]
```diff
@@ -376,6 +376,56 @@ teardown() {
     ! ls puck2/pyaml
 }
 
+@test "py2.7 can package module requirements with zip option" {
```
Comment (sbonami) [inline test.bats]: @dschep Is it expected functionality for the `zip` flag _not_ to move up the root `module`?

Comment (dschep) [reply test.bats]: I don't think so. That seems to be the cause of #203

Comment (felipe-augusto) [reply test.bats]: I managed to fix this problem, doing this: https://github.com/felipe-augusto/serverless-python-requirements/commit/2a14d9246eb8cb9e2e3b8e3ae2c5bc66a4f8e6b7

But i don't know if this is the desired behavior.

Comment (sbonami) [reply test.bats]: @dschep I believe so as well.

To anyone: Feel free to cherry-pick or modify the regression tests from 6f7b229 and a20a209 to validate the fix. I believe a fix to that issue precedes my addition.

**Pull request closed without merging**